### PR TITLE
Add slack bot for release notification on profiles channel

### PR DESCRIPTION
**Pull request by Skarlso**
### Description

Closes https://github.com/weaveworks/profiles/issues/249

### Checklist
- [ ] Added tests that cover your change
- [ ] Added/modified documentation as required (such as the `README.md`)
- [ ] Made sure the title of the PR is a good description that can go into the release notes
- [ ] Added a `kind` label to the PR (e.g. `kind/feature`)

### CODE REVIEW

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -67,4 +67,17 @@ jobs:
...
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
\ No newline at end of file
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Get release doc content
+        id: vars
+        run: echo ::set-output name=release_notes::$(cat docs/release_notes/${{ steps.prep.outputs.VERSION }}.md)
+      - name: Notify slack of new release
+        env:
+          SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN }}
```
Comment (aclevername) [inline .github/workflows/release.yaml:76]: does this exist?

Comment (Skarlso) [reply .github/workflows/release.yaml:76]: https://github.com/weaveworks/corp/issues/1911

Comment (aclevername) [reply .github/workflows/release.yaml:76]: shall we block merging on this? I'll pre-approve now.

**Review by aclevername — APPROVED** [review]

File: .github/workflows/release.yaml (hunk 1 above)
Comment (Skarlso) [reply .github/workflows/release.yaml:76]: Yeah for sure. I'll wait until that exists.

**Pull request merged**